src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -225,8 +220,26 @@ public function has($id)
      */
     public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE)
     {
+        $serviceNotFound = function ($id) use ($invalidBehavior) {
```

Comment:
I think there is no need for any levenstein reco for private services. There is no typo there.